### ISSUE TITLE
Accessibility issues - Add Dataset

### ISSUE THREAD

**Issue by alyce** [magda-io/magda#2267]
6 accessibility issues and 2 warnings identified in the two Add Dataset flow so far which need to be rectified.

Please see Google doc for full description: [https://docs.google.com/document/d/1Wcy8CQ_lugtasoB40Et8dF3S2j_64PkRun4DLdKP4RQ/edit#] 

Issue 1: Alt text missing
Issue 2: H1 missing
Issue 3: Colour contrast of progress meter fails
Issue 4: Text size defined in pixels, not relative terms
Issue 5: Multiple navigation paths not supported
Issue 6: No review and confirm in Add Dataset flow
Warning 1: Link labels for buttons
Warning 2: Required fields not displayed
